Routes/VendorAppApi.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/', function(req, res) {
+    var vendorAppData = {
+        VendorId: "UUID",
+        VendorAuth: "Password",
+        Components: [
+            {
+                Component: "form",
+                Fields: [
+                    {
+                        Child: "textbox",
+                        label: "Full name",
+                        Validate: true,  // optional
+                        Sensitive: true // optional
+                    },
+                    {
+                        Child: "phoneNumber",
+                        label: "Phone number"
+                    },
+                    {
+                        Child: "date",
+                        label: "Birthday"
+                    },
+                    {
+                        Child: "textbox",
+                        label: "Driver’s license number"
+                    },
+                ]
+            },
+            {
+                Component: "card",
+                Header: "header text",
+                body: "inner text",
+                Footer: "footer text"
+            },
+            {
+                Component: "button",
+                Text: "inner text",
+                "Submit location": "url"
+                //Callback: STRETCH GOAL
+            },
+            {
+                Component: "link",
+                Src: "url",
+                Alt: "alt text"
+            },
+            {
+                Component: "image",
+                Src: "url",
+                Alt: "alt text"
+            },
+            {
+                Component: "text",
+                Content: "I am text",
+                Label: "text label"  // optional
+            },
+            {
+                Component: "table",
+                display: 5,
+                headers: [    // optional
+                    "Header 1"
+                ],
+                rows: [    
+                    {
+                        Component: "text",
+                        Content: "I am a table cell",
+                    },
+                    {
+                        Component: "image",
+                        Src: "url",
+                        Alt: "table image"
+                    }
+                ]
+            }
```

Comment:
just to confirm this sample data is never used.

Reply:
Right, this stuff isn't being used anywhere because we don't have any frontend components for these things